shared-helpers/src/utilities/pdfs.ts

```diff
@@ -1,18 +1,20 @@
 import { ListingEvent, ListingEventType } from "@bloom-housing/backend-core/types"
+import { CloudinaryFileService } from "@bloom-housing/shared-services"
 
-export const cloudinaryPdfFromId = (publicId: string, cloudName: string) => {
+export const cloudinaryPdfFromId = (publicId: string) => {
+  const cloudName = process.env.cloudinaryCloudName || process.env.CLOUDINARY_CLOUD_NAME
```

Comment:
I'm a big fan of reducing reliance on global variables.  I realize that the function signatures are different between `cloudinaryPdfFromId` and `cloudinaryUrlFromId` for some reason, but can we pass cloudName into the class constructor and then directly pass that into those functions rather than looking up the global each time?

Side note: Next.js made the absolutely bonkers design decision to override the runtime `process.env` with build-time values.  It makes an complete mess where process.env.WHATEVER can return different values in different sections of the same application running in the same process.  Injecting values explicitly can reduce the chance of odd behavior and makes it easier to replace them with something like `serverRuntimeConfig`.

Reply:
Different thing on my end -- can we use the `ConfigService` pattern here instead of referencing the env variable directly? Check out shared.module.ts and `this.configService.get<string>` in #17

Reply:
That would be even better, IMO.  I don't think we want to rely on `nestjs` to provide that functionality (as in the example above) since 1) it's a big library for a relatively small amount of functionality and 2) the frontend services would then have dependencies on both `nestjs` and `nextjs`, but I'm in favor of that pattern in general and use it almost exclusively in my own projects.  I think a POJO would work as a first step.

Reply:
@ifranch  Following up on our out-of-band convo I'm fine with omitting entirely until the other pieces are in place given that adding it to the constructor would create even more global var sprawl in the short term.

Reply:
@lisunshiny I did try adding the configService in the constructor but it's not as straightforward as in the listingsService example. Clients that use the listingsService get it from the AuthContext like here https://github.com/bloom-housing/bloom/blob/main/sites/public/src/pages/account/applications.tsx#L18 and AuthContext imports the services from this autogenerated file https://github.com/bloom-housing/bloom/blob/main/backend/core/types/src/backend-swagger.ts which does not apply for this new service which is not defined in the backend.

Reply:
To be clear, I think what Liann is mostly advocating for is the config pattern, not a specific implementation or library.  In this case you could just make a `CloudinaryConfig` interface with a single property named "cloudName" of type string and pass that into the service constructor

Reply:
Oh I think I mean to reimplement a config service in your dir, not to necessarily reuse the one that lives within backend/core (each dir seems to have its own way of doing config stuff....)

In any case none of this is blocking on my end either!

Reply:
I will add a config service in a future PR.